src/constants.js

```diff
@@ -24,23 +28,30 @@ const displayMedium = `500 ${displayFontSize}/${displayLineHeight} ${displayFont
 
 const brightGreen = "#25b894";
 const darkGreen = "#005450";
+const darkerGreen = "#00413E";
```

Comment:
I've seen the light, lighter, lightest, dark, darker, darkest pattern used elsewhere.  It has its obvious limitations so I'm open to suggestions for improvement if we don't want to go down this path.

Reply:
this is totally fine, i don't see the point in being fussy about this unless we are actually being given an explicit palette of named colors by Juan (which we are not) ... I think that in practice we should try to use the Theme object to define functional colors as much as possible to minimize the number of references to color names, since I would expect them to drift and become more nonsensical over time. 

Plus one of the things coming down the pike is co-branding, meaning we may have to swap out many of these functional colors per state ... so we should get in the habit of not using explicit color names outside the theme.

Reply:
Cool, cool.  I'll change this theme value to `footerBackground: darkerGreen;` to make it functional.